kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/ResourceTest.java

```diff
@@ -97,8 +97,8 @@ public void testWatch() throws InterruptedException {
         .withResourceVersion("1")
       .withNamespace("test").and().build();
 
-      server.expect().get().withPath("/api/v1/namespaces/test/pods").andReturn(200, pod1).once();
-      server.expect().post().withPath("/api/v1/namespaces/test/pods").andReturn(201, pod1).once();
+      server.expect().get().withPath("/api/v1/namespaces/test/pods/pod1").andReturn(200, pod1).once();
+      server.expect().post().withPath("/api/v1/namespaces/test/pods/pod1").andReturn(201, pod1).once();
```

Comment:
The post corresponds to a create method. To create a pod _(say pod1)_, you need to post to the /api/v1/namespaces/test/pods/